src/framework/ui/uitypes.h

```diff
@@ -187,9 +189,32 @@ enum class Checkable {
     Yes
 };
 
+enum class ActionCategory {
+    UNDEFINED = -1,
+    INTERNAL,
+    TABLATURE,
+    VIEWINGNAVIGATION,
+    PLAYBACK,
+    LAYOUTFORMATTING,
+    SELECTINGEDITING,
+    APPLICATION,
+    ACCESSIBILITY,
+    FILE,
+    SELECTIONNAVIGATION,
+    TEXTLYRICS,
+    CHORDSYMBOLSFIGUREDBASS,
+    MEASURES,
+    MUSICALSYMBOLS,
+    DIALOGSPANELS,
+    NOTEINPUT,
+    WORKSPACE,
+    PLUGINS
```

Comment:
Although there's some inconsistency within the project, I think in new code it is more common to use CamelCase. It has may preference, because it looks more friendly and I think it's better readable.

Also, in any case, the words should be separated, e.g. `VIEW_NAVIGATION` or `ViewNavigation`, `LAYOUT_FORMATTING` or `LayoutFormatting`, etc.